.circleci/config.yml

```diff
@@ -139,6 +150,8 @@ workflows:
       - clang-sse-dynamic
       - clang-sse-static
       - clang-sse-sanitize
+      - gcc-avx-threaded
+      - gcc-avx-unthreaded
```

Comment:
Do you suggest that doing the same on clang is not necessary?

Reply:
No, testing all combinations is good except for the increased test time. I added the clang variants!

Reply:
Thanks Paul.  Good work.